packages/storybook/stories/additional-docs.js

```diff
@@ -1,8 +1,7 @@
 import { category, level } from './maturity-scale';
 
 const { CAUTION, DONT_USE, USE } = category;
-const { AVAILABLE, BEST_PRACTICE, CANDIDATE, DEPLOYED, DEPRECATED, PROPOSED } =
-  level;
+const { AVAILABLE, BEST_PRACTICE, DEPLOYED, DEPRECATED } = level;
```

Comment:
Should this have changed? It looks like `CANDIDATE` and `PROPOSED` were removed.

Reply:
Those were unused so I thought to remove it.

Reply:
I think it's possible we could in the future again have candidate or proposed components based on the [maturity scale](https://design.va.gov/about/maturity-scale) so it might be worth keeping them in there for now.